src/Solr/Reindex/Jobs/SolrReindexQueuedJobBase.php

```diff
@@ -138,6 +138,11 @@ public function getJobType()
         return QueuedJob::QUEUED;
     }
 
+    public function getSignature()
```

Comment:
Not important to this review since it's pre-existing, but I feel like this should be named something more like `generateSignature`, since it's not fetching anything, and would return a different value every time, even when called on the same object.

Reply:
Yeah, this is a queuedjobs API unfortunately

Reply:
oic, welp, that's fine then :)